erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -191,6 +191,9 @@ def reset_use_multi_level_bom(self):
 			self.get_items_and_operations_from_bom()
 
 	def validate_workstation_type(self):
+		if self.docstatus != 1:
```

Comment:
```suggestion
		if not self.docstatus.is_submitted():
```

Just a suggestion from a passerby 😺